src/design/fonts.css

```
@@ -0,0 +1,29 @@
+/*
+ * Self-hosted OFL fonts - no runtime Google Fonts <link>, the product runs in
+ * offline homelabs. Sources: github.com/google/fonts/ofl/{unbounded,
+ * spacegrotesk,spacemono}, converted to WOFF2 with woff2_compress.
+ * Licences: fonts/OFL-Unbounded.txt, OFL-SpaceGrotesk.txt, OFL-SpaceMono.txt.
+ * Unbounded and Space Grotesk are variable (wght axis), so one file covers
+ * every weight the design system uses.
+ */
+@font-face {
+  font-family: "Unbounded";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Unbounded" (font-family-name-quotes)

(font-family-name-quotes)